src/App.tsx

```diff
@@ -15,47 +15,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type SortType = 'NONE' | 'ALPHABET' | 'LENGTH';
+type IsReversed = boolean;
+type IsReset = boolean;
```

Comment:
You don't need to save a boolean type in a variable. Just use `boolean` directly